BREAKING.md

```diff
@@ -70,4 +71,11 @@ This section details the desktop browser, JavaScript framework, and mobile platf
 
 - `ion-picker` and `ion-picker-column` have been renamed to `ion-picker-legacy` and `ion-picker-legacy-column`, respectively. This change was made to accommodate the new inline picker component while allowing developers to continue to use the legacy picker during this migration period.
   - Only the component names have been changed. Usages such as `ion-picker` or `IonPicker` should be changed to `ion-picker-legacy` and `IonPickerLegacy`, respectively.
-  - Non-component usages such as `pickerController` or `useIonPicker` remain unchanged. The new picker displays inline with your page content and does not have equivalents for these non-component usages. 
\ No newline at end of file
+  - Non-component usages such as `pickerController` or `useIonPicker` remain unchanged. The new picker displays inline with your page content and does not have equivalents for these non-component usages. 
+
+<h4 id="version-8x-searchbar">Searchbar</h2>
```

Comment:
Shouldn't this be a `h3`?

Reply:
The component sub-levels are currently broken out as h4 elements: https://github.com/ionic-team/ionic-framework/blob/58c45515562b9c85cb2aa3dff198a903af95387f/BREAKING.md?plain=1#L51

The "Picker" heading was the only one that didn't follow the existing pattern & has been addressed here: https://github.com/ionic-team/ionic-framework/pull/28725. 

This aligns with the formatted we followed for v7: https://github.com/ionic-team/ionic-framework/blob/58c45515562b9c85cb2aa3dff198a903af95387f/BREAKING_ARCHIVE/v7.md?plain=1#L65-L67

Reply:
Hmm, but this would fail accessibility checks for headings.

Reply:
The axe devtools pass on the page(outside of a color contrast issue from GitHub). The heading ranks mentions:
> Skipping heading ranks can be confusing and should be avoided where possible

If you want to open that conversation to the entire team in Slack, I think that's great. That shouldn't block this work though.